content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -85,6 +85,11 @@ to manage the lifecycle of Istio.
     {{< text bash >}}
     $ helm init --service-account tiller
     {{< /text >}}
+    
+1. Install CRDs:
```

Comment:
This step isn't needed. It's already done in step 1 of the instructions, above.
https://deploy-preview-2252--preliminary-istio.netlify.com/docs/setup/kubernetes/helm-install/#installation-steps